lib/telephone_appointments/summary_document_activity.rb

```diff
@@ -0,0 +1,24 @@
+module TelephoneAppointments
+  class SummaryDocumentActivity
+    API_VERSION = 'v1'.freeze
+    PATH = 'summary_documents'.freeze
+
+    def initialize(appointment_id:, guider_uid:)
+      @appointment_id = appointment_id
+      @guider_uid = guider_uid
+    end
+
+    def save
+      @response = TelephoneAppointments.api.post(
+        "/api/#{API_VERSION}/#{PATH}",
```

Comment:
Technically speaking, all of this is the path. Is there much use in calling out the `API_VERSION` and `PATH` here? Instead of just putting the whole lot in a string without interpolation I mean.